rpcserver.go

```diff
@@ -1820,6 +1826,51 @@ func GetUnconfirmedBalance(w *wallet.Wallet, chainSvr *chain.Client, icmd interf
 	return (unconfirmed - confirmed).ToBTC(), nil
 }
 
+// ImportAddress handles an importaddress request by parsing
+// a encoded address and importing it as a watch-only address to the
+// imported address account.
+func ImportAddress(w *wallet.Wallet, chainSvr *chain.Client, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*btcjson.ImportAddressCmd)
+
+	addr, err := decodeAddress(cmd.Address, activeNet.Params)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = w.ImportAddress(addr, nil, *cmd.Rescan)
+	return nil, err
+}
+
+// ImportPubKey handles an importpubkey request by parsing a public key and
+// adding it to the imported account.
+func ImportPubKey(w *wallet.Wallet, chainSvr *chain.Client, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*btcjson.ImportPubKeyCmd)
+
+	pubKeyBytes, err := decodeHexStr(cmd.PubKey)
+	if err != nil {
+		return nil, err
+	}
+
+	var compressed bool
+	switch n := len(pubKeyBytes); n {
+	case btcec.PubKeyBytesLenUncompressed:
+		compressed = false
+	case btcec.PubKeyBytesLenCompressed:
+		compressed = true
+	default:
+		e := errors.New("Invalid serialized pubkey length")
```

Comment:
This error will be going away with the rest of the switch statement, but fwiw Go errors should not be capitalized (golint will complain).

The rpc server should probably capitalize all of them at a higher level though, to match the style used by core.